.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      SONAR_SCANNER_VERSION: 4.4.0.2170
```

Comment:
This can become outdated easily, no?
Maybe add a comment mentioning where to find the latest version?

Reply:
I expect it to be supported for at least a few years. I added a comment.